src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+function conversion(value) {
+  return +value
+    .split('').slice(1).join('').split(',').join('');
+}
+
+addEventListener('click', e => {
+  const sort = document.querySelector('thead').children[0].children;
+  const cheak = document.querySelector('tfoot');
+  const i = [...sort].findIndex(
+    element => element.innerText === e.target.innerText
+    && !cheak.contains(e.target));
+
+  if (i === -1) {
+    return;
+  }
+
+  const listValue = document.querySelector('tbody');
+
+  function sortList(list) {
```

Comment:
The sortList function is defined inside the event listener. Consider moving it outside of the event listener to avoid redeclaring it every time the event fires.